CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 Reactist follows [semantic versioning](https://semver.org/) and doesn't introduce breaking changes (API-wise) in minor or patch releases. However, the appearance of a component might change in a minor or patch release so keep an eye on redesigns and make sure your app still looks and feels like you expect it.
 
+## 3.1.3
+
+- [Tweak] Upgraded to latest storybook version
```

Comment:
I don't think we need a changelog entry for this as we won't cut a new version, right?

Reply:
Removed!